### ISSUE TITLE
MP GO:0000151     ubiquitin ligase complex

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14867]
GO:0000151     ubiquitin ligase complex
is not a child of "protein complex"

(although were we getting rid of protein complex in favour of macromolecular complex? I thought that was the plan at one point?)

**Comment by ukemi** [comment]
Yes. It is on my to-do list, but it is way more complicated than you might initially think because all the children need to be merged first. 

#12782

Macromolecular complex will become protein-containing complex and protein complex will be merged into it. We started to do it as part of Barbara's training and then realized that it was going to take too long to do as part of a training session. It is on the top of my priority list as part of the GHathon in February.

**Comment by bmeldal** [comment]
>but it is way more complicated than you might initially think

LOL! I initially made that request 5 years ago :)
